examples/preprocessors__typescript-browserify/cypress/integration/spec.ts

```diff
@@ -1,7 +1,5 @@
 import { add } from '../support/add'
 
-declare function expect(any):any
```

Comment:
I don't think this is needed anymore. The Cypress type definitions include `chai` and `mocha` definitions.

Reply:
good catch, should not be necessary anymore